Gemfile

```diff
@@ -1,9 +1,10 @@
 source 'https://rubygems.org'
 
-puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.3']
+ENV['PUPPET_VERSION'].nil? ? puppetversion = '~> 5.0' : puppetversion = ENV['PUPPET_VERSION'].to_s
 gem 'puppet', puppetversion
-gem 'puppetlabs_spec_helper', '>= 0.8.2'
-gem 'puppet-lint', '>= 1.0.0'
+gem 'puppetlabs_spec_helper', '>= 2.11.0'
+gem 'rspec-puppet-facts', '>= 1.8.0'
+gem 'metadata-json-lint'
```

Comment:
Without this gem explicitly declared, the `validate` rake task won't check the `metadata.json` file.